python/tank/util/metrics.py

```diff
@@ -42,6 +46,12 @@ class MetricsQueueSingleton(object):
 
     """
 
+    MAXIMUM_QUEUE_SIZE = 20
```

Comment:
this seems low to me. any reason for 20 specifically? can we bump it a bit to maybe 100 or so?

Reply:
The value of 20 is totally arbitrary, I'm actually missing a TODO: so we discuss on the maximum size.
 
I agree that 100 seems like a fair size, although if it was ever to queue up to 100 metric event, we probably want to ensure that it doesn't get sent in a single large payload. The server won't accept more than 10 at once. I would like to add a test to make sure it get broken in server-compatible payloads.